hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -82,40 +140,41 @@
         }
       ]
     },
-    { name: "ES_CTRL",
-      desc: "ENTROPY_SRC control register",
-      swaccess: "rw",
-      hwaccess: "hro",
-      regwen:   "ES_REGEN",
-      fields: [
-        { bits: "0",
-          name: "INIT_ENT_SRC",
-          desc: "Setting this bit will initialize the entropy source function."
-        }
-      ]
-    },
     { name:     "ES_STATUS",
       desc:     "ENTROPY_SRC status register",
       swaccess: "ro",
-      hwaccess: "hrw",
+      hwaccess: "hwo",
       hwext: "true",
       fields: [
-        { bits: "0",
-          name: "ENTROPY_RDY",
-          desc: '''This is set when the entropy source FIFO has met or exceeded the ES_THRESH value. This status bit will
-                automatically clear once the FIFO depth drops below the threshold value. This bit will not set unless the
-                block is enabled.
-                '''
+        { bits: "2:0",
+          name: "FIFO0_DEPTH",
+          desc: "This is the depth of the lfsr entropy source FIFO."
+        }
+        { bits: "5:3",
```

Comment:
nit: if you're going to space the DEPTH fields out, perhaps put them all spaced by 4 bits? (i.e. [2:0], [6:4], [10:8], [14:12])

Reply:
@sjgitty to answer question on documentation, the md doc has not been updated to reflect the changes. Need to do this, but was secondary to hjson and rtl updates for bronze. Will work on md doc if needed for this PR. 

Regarding the fifo depth bits, these bits moved around a lot when creating, so the alignment  ended up in an odd way - will fix.